seq2science/schemas/config/download.schema.yaml

```diff
@@ -28,13 +28,3 @@ properties:
   ascp_key:
     description: the public key of ascp
     type: string
-
-  # fastq-dump parameters
-  split:
-    description: fastq-dump parameters when splitting reads (paired-end)
-    default: --split-files --skip-technical --dumpbase --readids --clip --read-filter pass --defline-seq '@$ac.$si.$sg/$ri' --defline-qual '+'
-    type: string
-  splot:
-    description: fastq-dump parameters when splotting reads (single-end)
-    default: --split-spot  --skip-technical --dumpbase --readids --clip --read-filter pass --defline-seq '@$ac.$si.$sg/$ri' --defline-qual '+'
-    type: string
```

Comment:
question: so these are all done by default now? Their wiki only mentioned split-spot and skip-technical.

Reply:
Yes, fasterq-dump has "reasonable" defaults apparantly

Reply:
nice! just wanted to check :)